tests/queries/0_stateless/03229_query_condition_cache.sql

```diff
@@ -0,0 +1,40 @@
+-- Tags: no-parallel
+-- Tag no-parallel: Messes with internal cache
+
+SYSTEM DROP QUERY CONDITION CACHE;
```

Comment:
better not to drop table. usually we run query with specific `query_id` and then analyze its metrics: https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/02293_selected_rows_and_merges.sh